File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/formatters/test_helper_formatters.py
# test___init__.py – Tests helpers formatters (Python)
import importlib.util
import os
spec = importlib.util.spec_from_file_location(
    "helper_formatters",
    "/workspaces/dihya.io/backend/components/metiers/threed/fixtures/helpers/formatters/helper_formatters.py"
)
helper_formatters = importlib.util.module_from_spec(spec)
spec.loader.exec_module(helper_formatters)


def test_format_date():
    from datetime import datetime

    dt = datetime(2025, 6, 9, 12, 0, 0)
    assert helper_formatters.format_date(dt) == "2025-06-09T12:00:00"

File: backend/components/metiers/threed/tests_centralises/fixtures/helpers/formatters/test_helper_formatters.js
// test_helper_formatters.js – Tests helpers formatters (JS)
const { formatDate } = require('../../../../fixtures/helpers/formatters/helper_formatters');
test('formatDate', () => {
  expect(formatDate('2025-06-09T12:00:00')).toBe('2025-06-09T12:00:00.000Z');
});
